src/options.c

```diff
@@ -488,7 +488,7 @@ void display_help()
     puts( "      --autopoweroff      Power off system on completion of wipe delayed for" );
     puts( "                          for one minute. During this one minute delay you can" );
     puts( "                          abort the shutdown by typing sudo shutdown -c\n" );
-    puts( "      --sync=NUM          Will perform a sync after NUM writes (default: 0)" );
+    puts( "      --sync=NUM          Will perform a sync after NUM writes (default: 100000)" );
```

Comment:
maybe we could define a constant for `100000`?

Reply:
Sounds good, something like 'default_sync_rate'  or maybe 'sync_rate_100k' ?

Reply:
yeah `DEFAULT_SYNC_RATE` would make sense

Reply:
Sorry, missed the conversation totaly.
I've added a constant...